Formula/ooniprobe.rb

```diff
@@ -3,287 +3,146 @@ class Ooniprobe < Formula
-  end
-
-  resource "Werkzeug" do
-    url "https://files.pythonhosted.org/packages/13/a2/c4f2a1e1e9239cd979de00a2d7e0008559c46d920e9842e9b8063c5e6bf5/Werkzeug-0.12.tar.gz"
-    sha256 "f007848ed997101cb5c09a47e46c0b0b6f193d0f8a01cd2af920d77bf1ab4e68"
-  end
-
-  resource "zope.interface" do
-    url "https://files.pythonhosted.org/packages/44/af/cea1e18bc0d3be0e0824762d3236f0e61088eeed75287e7b854d65ec9916/zope.interface-4.3.3.tar.gz"
-    sha256 "8780ef68ca8c3fe1abb30c058a59015129d6e04a6b02c2e56b9c7de6078dfa88"
-  end
+  depends_on "go@1.14" => :build
 
   def install
-    # provided by libdnet
-    inreplace "requirements.txt", "pydumbnet", ""
-
-    # force a distutils install
-    inreplace "setup.py", "def run(", "def norun("
-
-    # obey the settings.ini we write
-    inreplace "ooni/settings.py", /(IS_VIRTUALENV = ).*/, "\\1 False"
-
-    (buildpath/"ooni/settings.ini").atomic_write <<~EOS
-      [directories]
-      usr_share = #{pkgshare}
-      var_lib = #{var}/lib/ooni
-      etc = #{etc}/ooni
-    EOS
-
-    ENV.append "CPPFLAGS", "-I#{MacOS.sdk_path}/usr/include/ffi" if MacOS.sdk_path_if_needed
+    ENV["GOPATH"] = HOMEBREW_CACHE/"go_cache"
+    (buildpath/"src/github.com/ooni/probe-cli").install buildpath.children
 
-    virtualenv_install_with_resources
+    cd "src/github.com/ooni/probe-cli" do
+      system "go", "build", "-o", bin/"ooniprobe", "./cmd/ooniprobe"
 
-    man1.install Dir["data/*.1"]
-    pkgshare.install Dir["data/*"]
-  end
-
-  def post_install
-    return if (pkgshare/"decks-available").exist?
-
-    ln_s pkgshare/"decks", pkgshare/"decks-available"
-    ln_s pkgshare/"decks/web.yaml", pkgshare/"current.deck"
-  end
+      prefix.install_metafiles
+    end
 
-  def caveats
-    <<~EOS
-      Decks are installed to #{opt_pkgshare}/decks.
+    (pkgshare/"etc/ooniprobe-daily-config.json").write <<~EOS
+    {
+      "_version": 3,
+      "_informed_consent": true,
+      "sharing": {
+        "include_ip": false,
+        "include_asn": true,
+        "upload_results": true
+      },
+      "nettests": {
+        "websites_url_limit": 100,
+        "websites_enabled_category_codes": [
+          "ALDR",
+          "ANON",
+          "COMM",
+          "COMT",
+          "CTRL",
+          "CULTR",
+          "DATE",
+          "ECON",
+          "ENV",
+          "FILE",
+          "GAME",
+          "GMB",
+          "GOVT",
+          "GRP",
+          "HACK",
+          "HATE",
+          "HOST",
+          "HUMR",
+          "IGO",
+          "LGBT",
+          "MILX",
+          "MISC",
+          "MMED",
+          "NEWS",
+          "POLR",
+          "PORN",
+          "PROV",
+          "PUBH",
+          "REL",
+          "SRCH",
+          "XED"
+        ]
+      },
+      "advanced": {
+        "use_domain_fronting": false,
+        "send_crash_reports": true,
+        "collect_usage_stats": true,
+        "collector_url": "",
+        "bouncer_url": "https://bouncer.ooni.io"
```

Comment:
I'd rather not add here any bouncer or collector URL. I think they're not supported anymore and in any case we want `ooniprobe` to automatically choose. For the same reason I would remove the domain fronting key.

Reply:
fixed!